docs/profiles.md

```diff
@@ -301,3 +304,35 @@ The following command runs the tests and applies the secrets specified in `profi
     $ inspec exec examples/profile-attribute --attrs examples/profile-attribute.yml
 
 See the full example in the InSpec open source repository: https://github.com/chef/inspec/tree/master/examples/profile-attribute
+
+# Profile files
+
+An InSpec profile may contain additional files that can be accessed during tests. This coveres use-cases where e.g. a list of ports is provided to be tested.
+
+To access these files, they must be stored in the `files` directory at the root of a profile. They are accessed by their name relative to this folder with `inspec.profile.file(...)`.
+
+Here is an example for reading and testing a list of ports. The folder structure is:
```

Comment:
Totally your call. I'd be pretty frustrated if I came across a profile that had values like that in a separate file... just feels like unnecessary abstraction.

If those values were used in 3+ different control files, then sign me up, it sounds great. If it's just in one, I don't understand the benefit.